sqlconnect/internal/databricks/integration_test.go

```diff
@@ -49,6 +49,31 @@ func TestDatabricksDB(t *testing.T) {
 		)
 	})
 
+	t.Run("with oauth", func(t *testing.T) {
+		oauthConfigJSON, ok := os.LookupEnv("DATABRICKS_OAUTH_TEST_ENVIRONMENT_CREDENTIALS")
```

Comment:
Are these credentials store somewhere in notion? `DATABRICKS_OAUTH_TEST_ENVIRONMENT_CREDENTIALS`

Reply:
No, I had created it yesterday for the POC. Added the same in Github.